app/app/urls.py

```diff
@@ -49,6 +49,11 @@
     url(r'^api/v0.1/', include(ebrouter.urls)),
     url(r'^actions/api/v0.1/', include(dbrouter.urls)),  # same as active, but not cached in cluodfront
 
+    # grant views
+    path('grant/<int:grant_id>/', dashboard.views.grant_show, name='grant'),
+    url(r'^grants/new?', dashboard.views.new_grant, name='new_grant'),
+    url(r'^grants/?', dashboard.views.grants, name='grants'),
```

Comment:
Can you modify these two `url` entries to the new `re_path`? Should be a matter of just replacing `url(` with `re_path(`